panel/template/fast/grid/fast_grid_template.css

```diff
@@ -124,7 +124,7 @@ img.app-logo {
 .sidenav fast-menu {
   width: 100%;
 }
-.sidenav a {
+.sidenav fast-menu a {
```

Comment:
This css was originally added to support creating a HTML based expandable menu like the one previously at awesome-panel.org.